src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder.java

```diff
@@ -403,7 +401,7 @@ public boolean perform(AbstractBuild build, Launcher launcher, BuildListener lis
         EnvVars env = build.getEnvironment(listener);
         Map<String, String> parameters = build.getBuildVariables();
 
-        log = listener.getLogger();
+        final PrintStream log = listener.getLogger();
```

Comment:
Oh, this is not just a JEP-200 issue—this could have blown up in nasty ways anyway, given concurrent builds.